framework/include/utils/MooseRandom.h

```diff
@@ -167,6 +167,11 @@ class MooseRandom
       it->second.first = it->second.second;
   }
 
+  LIBMESH_BEST_UNORDERED_MAP<unsigned int, std::pair<mt_state, mt_state> > getStates() { return _states; }
```

Comment:
Oh, can it be set to different things? Yes, it's `std::unordered_map` on my machine. Is a single helper for `LIBMESH_BEST_UNORDERED_MAP` sufficient or is a helper needed for all possible settings? Or is `LIBMESH_BEST_UNORDERED_MAP` the wrong choice in MooseRandom.h?